src/fgimic/jfield.f90

```diff
@@ -674,6 +701,66 @@ subroutine get_property(this)
           tmp = (sigma(1) + sigma(2) + sigma(3))/3.0d0
           write(*,*) "shielding constant sigma in ppm= ", tmp
 
+          if(elements_exists) then
+              if (k < 10) then
```

Comment:
that's one level of indentation too much

Reply:
ok